### PR TITLE
Crop large text data instead of dropping result

### PR DESCRIPTION
**Pull request by hluk**
JIRA: FACTORY-5780
Signed-off-by: Lukas Holecek <hluk@email.cz>

### CODE REVIEW

**Comment by hluk** [comment]
@lcarva @thrix  This is what we talked on the gating stakeholder meeting. Please review.

File: resultsdbupdater/utils.py [hunk 1]
```diff
@@ -13,21 +13,17 @@
...
     Even thought size for result data are not limited in database schema,
     Postgresql index has a size limited ("Values larger than 1/3 of a buffer
     page cannot be indexed").
     """
     for k, v in data.items():
         if isinstance(v, str) and len(v) > MAX_RESULT_DATA_SIZE:
-            raise exceptions.InvalidMessageError(
-                'Value for key "{0}" is too large (maximum size is {1})'.format(
-                    k, MAX_RESULT_DATA_SIZE))
+            data[k] = v[:MAX_RESULT_DATA_SIZE]
```
Comment (lcarva) [inline resultsdbupdater/utils.py]: Can we log that this is happening? Ideally, also include the message ID.

Comment (thrix) [reply resultsdbupdater/utils.py]: +1 for logging this

Comment (hluk) [reply resultsdbupdater/utils.py]: I've added logging and also ellipsis to the end of cropped text.

Comment (hluk) [reply resultsdbupdater/utils.py]: BTW, the `msg.log` is set up to include the message ID.

**Pull request merged**